Python/Indentation Rules.tmPreferences

```diff
@@ -6,7 +6,7 @@
 	<key>settings</key>
 	<dict>
 		<key>decreaseIndentPattern</key>
-		<string>^\s*(elif|else|except|finally)\b.*:</string>
```

Comment:
Should probably disable these patterns for multi-line `string`s if we're making them less specific. For comments, the Default package already exempts them from indentation rules by setting `preserveIndent

Reply:
Using jwortmanns approch for Matlab keeping `.*:` as is.